### PR TITLE
Cypress test. Tag annotation mode

### PR DESCRIPTION
**Pull request by DmitriyOparin**
Cypress test. Tag annotation mode
Add className for elements

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx [hunk 1]
```diff
@@ -245,11 +245,12 @@ function TagAnnotationSidebar(props: StateToProps & DispatchToProps): JSX.Elemen
                         </Checkbox>
                     </Col>
                 </Row>
-                <Row type='flex' justify='start'>
+                <Row type='flex' justify='start' className='cvat-tag-anntation-sidebar-frame-tags'>
```
Comment (bsekachev) [inline cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx:248]: Probably typos: cvat-tag-annotation-sidebar-frame-tags

File: cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx [hunk 2]
```diff
@@ -245,11 +245,12 @@ function TagAnnotationSidebar(props: StateToProps & DispatchToProps): JSX.Elemen
...
                     <Col>
                         <Text strong>Frame tags:&nbsp;</Text>
                         {frameTags.map((tag: any) => (
                             <Tag
+                                className={'cvat-tag-anntation-sidebar-frame-tag-label'}
```
Comment (bsekachev) [inline cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx:253]: The same typos

File: tests/cypress/support/commands.js [hunk 3]
```diff
@@ -490,3 +490,19 @@ Cypress.Commands.add('writeFilterValue', (clear, filterValue) => {
...
         });
 });
+
+Cypress.Commands.add('checkFrameNum', (frameNum) => {
+    cy.get('.cvat-player-frame-selector').within(() => {
+        cy.get('input[role="spinbutton"]').should('have.value', frameNum);
+    });
+});
+
+Cypress.Commands.add('goToNextDefaultFrame', (expectedFrameNum) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:500]: why "default"? Just ``goToNextFrame``. 
Haven't we had such the command till the present time?

File: cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx (hunk 1 above)
Comment (DmitriyOparin) [reply cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx]: Typo fixed.
These taken from [another element](https://github.com/openvinotoolkit/cvat/blob/develop/cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx#L236)  whith contains typo. Do I need to fix this?

File: cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx (hunk 2 above)
Comment (DmitriyOparin) [reply cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx]: Typo fixed.

File: tests/cypress/support/commands.js (hunk 3 above)
Comment (DmitriyOparin) [reply tests/cypress/support/commands.js]: Function renamed. This function did not use in global file (commands.js).

File: cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx (hunk 1 above)
Comment (DmitriyOparin) [reply cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx]: Typo "cvat-tag-anntation-sidebar-checkbox-skip-frame" fixed on "cvat-tag-annotation-sidebar-checkbox-skip-frame".

**Review by bsekachev — DISMISSED** [review]

**Comment by bsekachev** [comment]
@DmitriyOparin There are a couple of conflicts in the patch

**Review by bsekachev — APPROVED** [review]

**Pull request merged**